Set the hostname from the inventory file

jbradberry opened the pull request.

jbradberry committed:
Set the hostname from the inventory file

copelco commented:
Sometimes I've run into ``unable to resolve host`` errors when using ``sudo`` without also setting ``/etc/hosts`` to match: https://askubuntu.com/questions/59458/error-message-sudo-unable-to-resolve-host-user

copelco commented on README.rst:
```diff
@@ -61,6 +61,26 @@ The project then should have access to the ``tequila-common`` role in
+on the name you gave it in your inventory file.  For example, if your
+inventory file consists of ::
+
+    web1 ansible_host=42.42.42.42
```
I recommend we suggest something like ``myprojname-web1``/``myprojname-web2`` in the example. External services like papertrail, New Relic, etc., use hostnames to identify the servers and I'd like to proactively namespace hosts so we avoid getting multiple generic ``web1``/``web2`` hosts reporting to these services.

jbradberry committed:
Suggest a more namespaced host name in the example

jbradberry committed:
Also add the hostname to /etc/hosts

jbradberry committed:
Update the readme

to note which files receive the changes for the hostname, and that a
reboot will be needed for the changes to take effect.

copelco commented:
👍
